UefiCpuPkg/Library/BaseRiscV64CpuExceptionHandlerLib/BaseRiscV64CpuExceptionHandlerSecLib.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+# RISC-V CPU Exception Handler Library for SEC

```

Comment:
I don't think we can name this as Base library if it is specific to SEC. It is better to move this under the main CpuExceptionHandlerLib itself similar to loongarch and add under SecPeiCpuExceptionHandlerLib.inf and DxeCpuExceptionHandlerLib.inf separately.